core/src/main/java/org/infinispan/remoting/rpc/RpcManager.java

```diff
@@ -48,6 +48,26 @@
    Map<Address, Response> invokeRemotely(Map<Address, ReplicableCommand> rpcs, RpcOptions options);
 
    /**
+    * Asynchronously sends the {@link ReplicableCommand} to the destination using the specified {@link DeliverOrder}.
+    *
+    * @param destination  the destination's {@link Address}.
+    * @param command      the {@link ReplicableCommand} to send.
+    * @param deliverOrder the {@link DeliverOrder} to use.
+    */
+   void sendTo(Address destination, ReplicableCommand command, DeliverOrder deliverOrder);
```

Comment:
Speaking of the `invokeRemotely()` logic, have you thought whether we should change the RPC stats to consider the duration of the RPC from the moment we sent the command to the primary and until we got the ack from the last backup? The same for the replication timeout, should it apply just to the interaction with the primary or should it include the time spent waiting for backup acks?